README.md

```diff
@@ -71,7 +69,6 @@ Clone this repository `git clone https://github.com/Azure-Samples/ms-identity-ja
 1. Open the [.authConfig.js](./App/authConfig.js) file and provide the required configuration values.
     1. Replace the string `Enter_the_Application_Id_Here` with your app/client ID on Azure AD portal.
     1. Replace the string `Enter_the_Tenant_Info_Here` with your tenant ID on Azure AD portal.
-    1. Replace the string `Enter_the_Redirect_Uri_Here` with the default redirect uri generated by portal `msal{Your_Application/Client_Id}://auth`
     1. Replace the string `Enter_the_Cloud_Instance_Id_Here` with `https://login.microsoftonline.com/` (see **note** below).
     1. Replace the string `Enter_the_Graph_Endpoint_Here`. with `https://graph.microsoft.com/` (see **note** below).
 
```

Comment:
I'll move this to a section called 

Sovereign clouds

and just place the links

Reply:
We are planning to remove this entirely because msal{Your_Application/Client_Id}://auth uses chromium, not a browser, and this is not recommended based on the OAuth2 spec for native apps.